core/src/components/select/select.tsx

```diff
@@ -863,15 +864,13 @@ export class Select implements ComponentInterface {
    * next to the select text.
    */
   private renderSelectIcon() {
-    const theme = getIonTheme(this);
-    const { isExpanded, toggleIcon, expandedIcon } = this;
+    const { isExpanded, selectExpandIcon, selectCollapsedIcon } = this;
     let icon: string;
 
-    if (isExpanded && expandedIcon !== undefined) {
-      icon = expandedIcon;
+    if (isExpanded) {
+      icon = selectExpandIcon;
     } else {
-      const defaultIcon = theme === 'ios' ? chevronExpand : caretDownSharp;
-      icon = toggleIcon ?? defaultIcon;
+      icon = selectCollapsedIcon;
```

Comment:
Minor, but should this be changed and combined with line 868?

```
let icon = isExpanded ? selectExpandIcon : selectCollapsedIcon;
```

Reply:
https://github.com/ionic-team/ionic-framework/pull/29373/commits/14cf2edbbc160beefdff5f0f65436cc24f7a2a32